truss/contexts/image_builder/serving_image_builder.py

```diff
@@ -312,6 +312,12 @@ def generate_docker_server_nginx_config(build_dir, config):
     assert (
         config.docker_server.server_port is not None
     ), "docker_server.server_port is required to use custom server"
+    assert (
+        config.docker_server.readiness_endpoint is not None
+    ), "docker_server.readiness_endpoint is required to use custom server"
+    assert (
+        config.docker_server.liveness_endpoint is not None
+    ), "docker_server.liveness_endpoint is required to use custom server"
```

Comment:
These should probably be exepctions...

Reply:
What do you mean?

Reply:
Assertions are primarily used as a debugging aid. They are meant to catch programming errors during development by verifying assumptions made by the programmer.

Exceptions are used for handling errors and other "exceptional" conditions that may arise at runtime due to external factors (e.g., invalid user input, file I/O, network errors).

Reply:
got it will change to exceptions in the next pr